### PR TITLE
Multi network

### PR DESCRIPTION
**Pull request by GabrielCamba**
Added gc and rinkeby using mustache

Already deployed on https://thegraph.com/hosted-service/subgraph/gnosis/cow-rinkeby and https://thegraph.com/hosted-service/subgraph/gnosis/cow-gc.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,11 +4,18 @@
...
-    "deploy": "graph deploy --node https://api.thegraph.com/deploy/ gnosis/cow",
-    "deploy:dev": "graph deploy --node https://api.thegraph.com/deploy/ <YourGithubUser/YourSubgraph>"
+    "prepare:mainnet": "mustache config/mainnet.json subgraph.yaml.mustache > subgraph.yaml",
+    "prepare:rinkeby": "mustache config/rinkeby.json subgraph.yaml.mustache > subgraph.yaml",
+    "prepare:gc": "mustache config/gc.json subgraph.yaml.mustache > subgraph.yaml",
+    "deploy": "yarn deploy:mainnet",
+    "deploy:mainnet": "yarn prepare:mainnet && yarn codegen && graph deploy gnosis/cow --node https://api.thegraph.com/deploy/",
+    "deploy:rinkeby": "yarn prepare:rinkeby && yarn codegen && graph deploy gnosis/cow-rinkeby --node https://api.thegraph.com/deploy/",
+    "deploy:gc": "yarn prepare:gc && yarn codegen && graph deploy gnosis/cow-gc --node https://api.thegraph.com/deploy/",
+    "deploy:dev": "yarn prepare:<Network> && yarn codegen && graph <YourGithubUser/YourSubgraph> --node https://api.thegraph.com/deploy/"
```
Comment (alfetopito) [inline package.json:14]: Do you think there's a use case for a single command that prepares and deploys on all networks?

Not that familiar with TheGraph so maybe deploying several sub-graphs at the same time doesn't make sense, just wondering.

**Review by alfetopito — APPROVED** [review]
Looks good, only one minor question/comment

File: package.json (hunk 1 above)
Comment (GabrielCamba) [reply package.json:14]: I never did it! I think you are proposing something like this:
```
"deploy:all": "yarn deploy:mainnet && yarn deploy:rinkeby && yarn deploy:gc",
```
I can imagine 1 bad scenario here. What if only the first deploy fails? Need to be very careful about that. Console output is verbose enough to make you not to notice there was an error.  The other thing is if you try to deploy a single subgraph and it fails maybe you should start working on what's failing before trying on another network.

Comment (alfetopito) [reply package.json:14]: Yes, that's what I had in mind.

Your comment makes sense, let's go without it

**Review by anxolin — COMMENTED** [review]
APPROVE


However, for another PR. Could you make sure instructions are clear in the README?
Please, try to make something very similar to this: https://github.com/gnosis/dex-subgraph

**Pull request merged**